lib/SILOptimizer/FunctionSignatureTransforms/ExistentialTransform.cpp

```diff
@@ -108,11 +108,7 @@ void ExistentialSpecializerCloner::cloneAndPopulateFunction() {
     // A return location can't be used for a non-return instruction.
     auto loc = RegularLocation::getAutoGeneratedLocation();
     for (SILValue cleanupVal : CleanupValues) {
-      if (cleanupVal.getOwnershipKind() == OwnershipKind::Guaranteed) {
-        Builder.emitEndBorrowOperation(loc, cleanupVal);
```

Comment:
Actually, ignore the test case thing I mentioned before. What I was trying to say with the test cases is I want to understand why this endBorrowOperation was here (to make sure its truly dead) and so I was suggesting writing test cases that might trigger it. But from our conversation you mentioned just putting an assert and trying to the build the stdlib, I think that will be fine as well as long as we keep the assertion in.

Reply:
I tested stdlib with the assert, I did not see any cases needing an end_borrow.